doc/provisioning.md

```diff
@@ -84,28 +91,27 @@ At the time of writing, the currently implemented provider-specific placement di
   - Availability Zone: both the AWS and OpenStack providers support `zone=<zone>`, directing the provisioner to start an instance in the specified availability zone.
   - MAAS: `<hostname>` directs the MAAS provider to acquire the node with the specified hostname.
 
-Availability Zone Spread
-------------------------
+## Availability zones
 
-For Juju providers that know about Availability Zones, instances will be 
-automatically spread across the healthy availability zones to maximise service 
+For Juju providers that know about Availability zones, instances will be
```

Comment:
What are availability zones? Is this something I should just know or could it be defined/linked to a definition here?

Are these the same as cloud regions?

Reply:
I added a link to the Juju doc for AZs.